python/tk_desktop/desktop_engine_site_implementation.py

```diff
@@ -48,6 +49,78 @@ def destroy_engine(self):
         if self.msg_server is not None:
             self.msg_server.close()
 
+
+    ###########################################################################
+    # panel support (displayed as tabs)
+    def _init_tabs(self):
+        """
+        Create desktop tabs from the registered application panels, following
+        their specified relative positioning.
+        """
+        # make the "Apps" tab appear as a fake panel to fit in the sorting
+        # logic
+        apps_tab = {"callback":   self.desktop_window._register_apps_tab,
+                    "properties": {"position": 0}}
+
+        panels = [apps_tab] + list(self._engine.panels.itervalues())
```

Comment:
Rather than calling list(dict().itervalues()) you may as well just call dict().values(), since you're keeping the whole list in memory at that point anyway.